src/modules/partitionq/partitionq.conf

```diff
@@ -0,0 +1,255 @@
+# for root that uses 100% of the space and uses the filesystem defined by
+# defaultFileSystemType.
+#
+# Note: the EFI system partition is prepend automatically to the layout if
+# needed; the swap partition is appended to the layout if enabled (small of
+# suspend).
+#
+# Otherwise, the partition layout is defined as follow:
+#
+# partitionLayout:
+#     - name: "rootfs"
+#       type: "4f68bce3-e8cd-4db1-96e7-fbcaf984b709"
+#       filesystem: "ext4"
+#       mountPoint: "/"
+#       size: 20%
+#       minSize: 500M
+#       maxSize: 10G
+#       attributes: 0xffff000000000003
+#     - name: "home"
+#       type: "933ac7e1-2eb4-4f13-b844-0e14e2aef915"
+#       filesystem: "ext4"
+#       mountPoint: "/home"
+#       size: 3G
+#       minSize: 1.5G
+#       features:
+#         64bit: false
+#         casefold: true
+#     - name: "data"
+#       filesystem: "fat32"
+#       mountPoint: "/data"
+#       features:
+#         sector-size: 4096
+#         sectors-per-cluster: 128
+#       size: 100%
+#
+# There can be any number of partitions, each entry having the following attributes:
+#   - name: filesystem label
+#           and
+#           partition name (gpt only; since KPMCore 4.2.0)
+#   - uuid: partition uuid (optional parameter; gpt only; requires KPMCore >= 4.2.0)
+#   - type: partition type (optional parameter; gpt only; requires KPMCore >= 4.2.0)
+#   - attributes: partition attributes (optional parameter; gpt only; requires KPMCore >= 4.2.0)
+#   - filesystem: filesystem type (optional parameter)
+#       - if not set at all, treat as "unformatted"
+#       - if "unformatted", no filesystem will be created
+#       - if "unknown" (or an unknown FS name, like "elephant") then the
+#         default filesystem type, or the user's choice, will be applied instead
+#         of "unknown" (e.g. the user might pick ext4, or xfs).
+#   - mountPoint: partition mount point (optional parameter; not mounted if unset)
+#   - size: partition size in bytes (append 'K', 'M' or 'G' for KiB, MiB or GiB)
+#           or
+#           % of the available drive space if a '%' is appended to the value
+#   - minSize: minimum partition size (optional parameter)
+#   - maxSize: maximum partition size (optional parameter)
+#   - features: filesystem features (optional parameter; requires KPMCore >= 4.2.0)
+#       name: boolean or integer or string
+
+# Checking for available storage
+#
+# This overlaps with the setting of the same name in the welcome module's
+# requirements section. If nothing is set by the welcome module, this
+# value is used instead. It is still a problem if there is no required
+# size set at all, and the replace and resize options will not be offered
+# if no required size is set.
+#
+# The value is in Gibibytes (GiB).
+#
+# BIG FAT WARNING: except for OEM-phase-0 use, you should be using
+#                  the welcome module, **and** configure this value in
+#                  `welcome.conf`, not here.
+# requiredStorage: 3.5
```

Comment:
Usually the .conf for a QML references the corresponding widgets .conf, only adds QML specific settings, not duplicate all instructions

Reply:
Fixed in latest commit, have removed all the partitionq.conf file settings, but left the file in there with a to do message, as I am not sure yet what settings actually need to be in QML module itself, will remove it if no settings live in qml in the future